src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -581,35 +572,34 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
             for (int i = 0; i < numOfRubricChoices; i++) {
                 String tableBodyCell =
                         Templates.populateTemplate(tableBodyFragmentTemplate,
-                                "${percentageFrequencyOrAverage}", df.format(rubricStats[j][i] * 100) + "%"
+                                Slots.RUBRIC_PERCENTAGE_FREQUENCY_OR_AVERAGE, df.format(rubricStats[j][i] * 100) + "%"
                                                                    + " (" + responseFrequency[j][i] + ")");
```

Comment:
Might be better if everything from `df.*` is on this line. I think it should fit without exceeding the line length limit.